### ISSUE TITLE
Unable to follow Ingress with NGINX task in Katacoda

### ISSUE THREAD

**Issue by benjamintanweihao** [kubernetes/website#27935]
This doesn't work with the launched version of MiniKube (1.8).,

Specifically, it fails when running the following command:

```shell
kubectl apply -f https://k8s.io/examples/service/networking/example-ingress.yaml
```

Returns:

no matches for kind "Ingress" in version "networking.k8s.io/v1"

**Comment by sftim** [comment]
/label team/katacoda
(other folks can help too)

If this Katacoda environment could use a newer version of Kubernetes, I think that'd fix it.

**Comment by sftim** [comment]
/retitle Unable to follow Ingress with NGINX task in Katacoda

**Comment by sftim** [comment]
/triage duplicate
First reported in #27493

**Comment by sftim** [comment]
Fixed by #29616
/close
